.travis.yml

```diff
@@ -9,9 +9,10 @@ osx_image: xcode6.4
 env:
   matrix:
     
-    - CONDA_NPY=113  CONDA_PY=27
-    - CONDA_NPY=113  CONDA_PY=35
-    - CONDA_NPY=113  CONDA_PY=36
+    - CONDA_NPY=107  CONDA_PY=27
+    - CONDA_NPY=109  CONDA_PY=35
+    - CONDA_NPY=109  CONDA_PY=34
+    - CONDA_NPY=111  CONDA_PY=36
```

Comment:
Yep. `conda-smithy` will do the right thing and "outsource" the `numpy` selection to `conda` via the different build/run pinning.

We are looking into doing the same for all the pinnings in conda-forge BTW, but this one was first b/c it reflects in the CIs directly. Other others will alleviate incompatibilities. The rule of thumb is to  build with the oldest version possible and run with the latest version that is backwards compatible. When a new version breaks the `ABI` we need to choose to use it as the base to build or take some time to adopt it. That choice is better than the current pinning scheme that we have which cause many breakages with `defaults`.